codeland.html

```diff
@@ -621,6 +621,13 @@
     document.querySelector('.codeland-livestream__speaker-info div p').innerHTML = talkData.bio;
     document.querySelector('.codeland-livestream__speaker-info div button').dataset.url = talkData.url;
 
+
+    if (talkData.sponsor) {
+      document.querySelector('.codeland-livestream__speaker-info__footer button').textContent = 'Learn more';
+    } else {
+      document.querySelector('.codeland-livestream__speaker-info__footer button').textContent = 'See discussion';
+    }
```

Comment:
Adding the url to the button like this:

```
if (talkData.sponsor) {
  document.querySelector('.codeland-livestream__speaker-info__footer button').textContent = 'Learn more';
  document.querySelector('.codeland-livestream__speaker-info div button').dataset.url = talkData.url;
} else {
  document.querySelector('.codeland-livestream__speaker-info__footer button').textContent = 'See discussion';
  document.querySelector('.codeland-livestream__speaker-info div button').dataset.url = `${talkData.url}#comments`;
}
```

And we would also need to change the [EventListener here](https://github.com/thepracticaldev/codeland-copy/blob/master/codeland.html#L1181) because the `#comments` would now be added directly to the element's `href`, making it look like this:

```
livestreamButt.addEventListener('click', () => {
    presentSidecar(
      livestreamButt.dataset.url,
      'Discussion Article',
      'Find all these articles in <a href="/t/codeland" target="_blank">#codeland</a>'
    );
  });
```

Reply:
I'm pretty sure sponsor links will just go the the sponsor's booth (article) on DEV, not linking externally.

This code looks great, I'll incorporate it. Thanks!